components/HomeSection/HomeSection.module.scss

```
@@ -10,33 +10,41 @@
     margin: 0 auto;
     max-width: 1080px;
     padding: 0 1rem 1rem;
-    position: relative;
-    z-index: 2;
   }
 }
 
 .container {
   margin: 0 auto;
-  max-width: 1080px;
+  max-width: 67.5rem;
```

We're dealing with viewport width here, pixels is easier to understand as that is how monitors resolutions are measured.

```suggestion
  max-width: 1080px;
```